robottelo/ui/contentsearch.py

```diff
@@ -0,0 +1,146 @@
+# -*- encoding: utf-8 -*-
+
+"""Implements Content Search UI"""
+
+from robottelo.ui.base import Base, UIError
+from robottelo.ui.locators import locators
+
+
+class ContentSearch(Base):
+    """Manipulates Content Search from UI"""
+
+    ft_locator = None
+
+    def select_filter_type(self, filter_type):
+        """Specify content type that you like to choose from 'Content' dropdown
+        At that moment we support next types:
+        'Content Views'
+        'Products'
+        'Repositories'
+        'Packages'
+        'Puppet Modules'
+        In the same time, it is necessary to keep order like in list above
+        (Content Views->Products->Repositories->Packages, Puppet Modules)
+        because in Content Search UI the lower you go in list more elements
+        will be added to panel
```

Comment:
Do you mean something like this?

    """Specify a content type to select from the 'Content' menu.

    The following content types are currently supported:

    1. Content Views
    2. Products
    3. Repositories
    4. Packages
    5. Puppet Modules

    It is advisable to respect the list ordering as shown above, because
    when using the Content Search UI, proceeding lower in the list causes
    more UI elements to be added to the panel.

    """